consensus/istanbul/backend/backend.go

```diff
@@ -867,6 +878,23 @@ func (sb *Backend) retrieveValidatorConnSet() (map[common.Address]bool, error) {
 	return sb.cachedValidatorConnSet, nil
 }
 
+// retrieveCachedValidatorConnSet returns the most recently cached validator conn
```

Comment:
It seems like this function and `retrieveValidatorConnSet` can be merged into one function with an addition boolean parameter such as `updateAsynchronously`.